scripts/version.sh

```diff
@@ -32,8 +32,8 @@ REVISION=$(git rev-parse HEAD)$(if ! git diff --no-ext-diff --quiet --exit-code;
 PLATFORM=${GOOS}-${GOARCH}
 RELEASE=${PROG}.${PLATFORM}
 # hardcode versions unless set specifically
-KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.29.11}
-KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.29.11-rke2r1-build20241202}
+KUBERNETES_VERSION=${KUBERNETES_VERSION:-v1.29.12}
+KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.29.12-rke2r1-build20241211}}
```

Comment:
Typo is causing CI to fail:
```suggestion
KUBERNETES_IMAGE_TAG=${KUBERNETES_IMAGE_TAG:-v1.29.12-rke2r1-build20241211}
```

Reply:
done